assignments/array-methods.js

```diff
@@ -56,28 +56,69 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+for (let i = 0; i < runners.length; i++) {
+    fullName.push(runners[i].first_name + " " + runners[i].last_name);
+}
```

Comment:
This definitely works, the instructions say to use .forEach(), though. What would this look like using .forEach()? What about .map()?